src/eva/data/data_collections.py

```diff
@@ -230,6 +241,7 @@ def display_collections(self):
         self.logger.info('-'*80)
         self.logger.info(fcol.bold + 'Collections available: ' + fcol.end)
         for collection in self._collections.keys():
+            print(self._collections[collection])
```

Comment:
is this for debugging or intended to stay?

Reply:
Good catch. Thanks.